### ISSUE TITLE
Training Discrepancies for normal CNN and Deformable CNN

### ISSUE THREAD

**Issue by HenryJia** [felixlaumon/deform-conv#1]
On line 62 and 99 of https://github.com/felixlaumon/deform-conv/blob/master/scripts/scaled_mnist.py#L62

I noticed you're training deformable conv with rescaled iamges but normal mnist with normal images. This makes no sense

Secondly, on line 91 you're also loading the alredy trained normal CNN weights into the deformable CNN

All of this makes no sense and obviously puts the deformable conv at an advantage, unless there's something I'm missing here

**Comment by mariolew** [comment]
Agreed.

**Comment by XiongweiWu** [comment]
Agreed too. Author u can train regular and deformable versions separately on the same dataset, otherwise it makes no sense.

**Comment by ShiningRuby** [comment]
do you remember the accuracy of the deformable convolution with regular images?the test accuracy is just 88%,and i don't know why

**Comment by Jing-Luo** [comment]
Could not agree more.
